src/psyclone/psyad/utils.py

```diff
@@ -60,6 +60,24 @@ def node_is_active(node, active_variables):
     return False
 
 
+def node_is_active_names(node, active_variable_names):
```

Comment:
How about overloading `node_is_active` so that it takes a list of either `str` or `Symbol`? Similarly for `node_is_passive`.

Reply:
Actually, since these methods can be removed when #1655 is done, perhaps just a TODO saying that will do.

Reply:
I've updated them in any case. I'm not sure we will want to remove them though.

Reply:
Thanks, I think it's a lot nicer.